src/app/client/src/app/modules/groups/components/group-header/group-header.component.ts

```diff
@@ -97,21 +97,24 @@ export class GroupHeaderComponent implements OnInit, OnDestroy {
     this.handleFtuModal.emit(visibility);
   }
 
-  addTelemetry(id, extra?) {
-    this.groupService.addTelemetry({id, extra}, this.activatedRoute.snapshot, [], _.get(this.groupData, 'id'));
+   /**
+   * @description - To set the telemetry Intract event data
+   * @param  {} edata? - it's an object to specify the type and subtype of edata
+   */
+  addTelemetry(id, extra?, edata?) {
+    this.groupService.addTelemetry({id, extra, edata}, this.activatedRoute.snapshot, [], _.get(this.groupData, 'id'));
   }
-
   toggleModal(visibility = false, name?: string, eventName?: string) {
     this.showModal = visibility;
     this.groupService.emitMenuVisibility('group');
     this.name = name;
     switch (name) {
       case actions.DELETE:
-        this.addTelemetry('delete-group', {status: _.get(this.groupData, 'status')});
+        this.addTelemetry('delete-group', {status: _.get(this.groupData, 'status')}, {type: 'select-delete'});
         this.assignModalStrings(this.resourceService.frmelmnts.lbl.deleteGroup, this.resourceService.messages.imsg.m0082, '{groupName}');
```

Comment:
here as well. check the types u added. FYI @vinukumar-vs

Reply:
type is given by sai CC: @vinukumar-vs